rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/charts/CpusUsageLineChart.java

```diff
@@ -68,31 +69,36 @@ public double formatValue(double value) {
 
     @Override
     public void processResult(String result) {
-        JSONObject object = controller.parseJSON(result).isObject();
-        if (object != null) {
+        try {
```

Comment:
I don't understand why the try-catch blocks include all that code. If you wanted to catch the parsing exception 
why you don't do every time you have to parse:
```
try {
JSONObject object = controller.parseJSON(result).isObject();        
} catch (JSONException e) {
          throw new IllegalArgumentException("signinficant message", e);
}
```

Reply:
Again, I don't want to change the existing code, which I why I mention, please look at the PR with ignore whitespace

Reply:
@fviale sorry i didnt see the comment on slack.